### PR TITLE
Token manager copy

### PR DESCRIPTION
**Pull request by bpierre**
_Builds on https://github.com/aragon/aragon-apps/pull/764 (do not merge before)_
_Fixes https://github.com/aragon/aragon-apps/issues/509_

See also https://github.com/aragonone/product/issues/41#issuecomment-480219715

## Preview

Screenshot from 2019-04-05 15-58-36
Screenshot from 2019-04-05 15-59-31
Screenshot from 2019-04-05 15-59-45
Screenshot from 2019-04-05 16-00-00
Screenshot from 2019-04-05 16-00-06
Screenshot from 2019-04-05 16-00-15
Screenshot from 2019-04-05 16-00-43

### CODE REVIEW

File: apps/token-manager/app/src/screens/EmptyState.js [hunk 1]
```diff
@@ -7,9 +7,9 @@ const EmptyState = ({ onActivate }) => (
   <Main>
     <EmptyStateCard
       icon={<img src={emptyIcon} alt="" />}
-      title="There are no tokens assigned yet"
-      text="Assign tokens to get started."
-      actionText="Assign Tokens"
+      title="There are no tokens added yet"
+      text="Add tokens to get started."
+      actionText="Add Tokens"
```
Comment (sohkai) [inline apps/token-manager/app/src/screens/EmptyState.js]: @dizzypaty I wonder if we could use some language like

"There are no token holders yet"

"Add tokens to an address to get started"

Comment (dizzypaty) [reply apps/token-manager/app/src/screens/EmptyState.js]: Yes, that sounds better!

**Comment by john-light** [comment]
I'm wondering how this fixes https://github.com/aragon/aragon-apps/issues/509 when "Remove tokens" is still called that in the Token Manager but it's called "Burn token" in the Permissions app?

**Comment by sohkai** [comment]
@john-light Good point, re-opened #509 since it's more related to permissions metadata than renaming the action.

**Pull request merged**